src/lwc/modal/modal.js-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="modal">
+    <apiVersion>46.0</apiVersion>
```

Comment:
This should be api 49. (Or 50 if you're setting all class/components in the PR to 50). They all have to match at either 49 or 50.